lib/create-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var File = require('vinyl'),
+	fs = require('fs');
+
+/**
+ * Creates a vinyl file
+ * @function createFile
+ *
+ * @param {STRING}  filePath  relative path to file
+ * @param {STRING}  cwd  current working directory
+ * @param {STRING}  base  directory where we start looking for the file
+ * @param {STRING}  type  stream|null
+ */
+exports.createFile = function createFile (filePath, cwd, base, type) {
+  var contents;
+
+  if (type == 'stream') {
+    contents = fs.createReadStream(filePath);
+  } else {
+    contents = fs.readFileSync(filePath);
```

Comment:
wasn't using readfilesync
but, figured out why the console.logs didn't fire...was pointing to fs-promise

still not working though.

This is what I have:

```
    fs.readFile(filePath, function (err, data) {
      if (err) throw err;
      console.log('data');
      console.log(JSON.stringify(data));
      console.log(filePath);
        return new File({
          path: filePath,
          cwd: cwd,
          base: base,
          contents: data
        });
    });
```

but the file is returned in /test/main.js as undefined.